content/no-code.md

```diff
@@ -88,16 +87,13 @@ This page briefly introduces a standard to help you make your no-code website to
 
 ## Why Accessibility Matters
 
-People with disabilities use your no-code tool to create and consume websites.
+Your no-code tool users include people with disabilities. Your tool needs to be accessible to disabled content creators.
```

Comment:
I am good with this change. "disabled content creators" is a bit different from our traditional "... with disabilities". I am happy for us to get closer to the social model of disability, but just wanted to raise this in case we do want to remain consistent with people-first language.

Reply:
Correct. We are using both "people with disabilities" and "disabled people" now. We need to update our documentation on that. (sidenote: we are also using both in the videos.)